nohup: ignoring input
2024-12-10 11:40:28 - INFO - PMTfication starts...
2024-12-10 11:40:28 - INFO - The number of files in the subdirectory: 2
2024-12-10 11:40:28 - INFO - Using up to 1 workers.
2024-12-10 11:40:28 - ERROR - An error occurred: Neither 'truth' nor 'Truth' table exists in the source database.
Traceback (most recent call last):
  File "/lustre/hpc/icecube/cyan/factory/DOMification/PMTfier.py", line 301, in <module>
    main()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/PMTfier.py", line 287, in main
    pmtfier = PMTfier(
  File "/lustre/hpc/icecube/cyan/factory/DOMification/PMTfier.py", line 37, in __init__
    self.truth_table_name = self._get_truth_table_name_db(first_db_file)
  File "/lustre/hpc/icecube/cyan/factory/DOMification/PMTfier.py", line 79, in _get_truth_table_name_db
    raise ValueError("Neither 'truth' nor 'Truth' table exists in the source database.")
ValueError: Neither 'truth' nor 'Truth' table exists in the source database.
